rdf-prefix.el

```diff
@@ -2596,7 +2863,8 @@
     ("zem" . "http://s.zemanta.com/ns#")
     ("zoology" . "http://purl.org/NET/biol/zoology#")
     ("zoomaterms" . "http://rdf.ebi.ac.uk/vocabulary/zooma/")
-    ("zr" . "http://explain.z3950.org/dtd/2.0/")))
+    ("zr" . "http://explain.z3950.org/dtd/2.0/")
+))
```

Comment:
Could you update the script to keep the previous format with the closing parentheses on the same line?